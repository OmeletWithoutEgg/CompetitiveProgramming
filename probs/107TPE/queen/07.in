750
90876045 -129193544
85198654 -133402428
-21821616 45841591
-103498745 88607350
25343038 -53661636
-95603044 111624374
78956909 -136781317
48267960 -140208195
-87112976 47203215
70528278 -42741686
100048977 -63665237
-86152624 46361443
46084897 -42755938
-36114340 37565946
75334948 -44210864
101893225 -115781409
25843476 -53228020
-73816880 38614866
-3766882 87363027
92568776 -127674368
-69530175 36959189
-10438097 109486009
-31454882 39589695
-6889021 66749859
8494225 -95221033
45495027 -42908829
-6585667 101182385
-6132096 68906915
105772420 -107033289
-12615645 112908596
105224742 -73872998
42740565 -43726123
-83735768 124336129
-55889945 134330712
-4195241 77204958
-8811226 106447538
-62676713 133563201
32555087 -48390492
-54905838 34394755
-68572513 132109615
-89504320 49501880
-103664629 85527172
-16691619 50734129
-68725537 132061592
16792427 -119057030
8380167 -93360261
-7950191 104603072
-29066031 40856564
-54140274 134377533
72831963 -139084311
50149546 -140557559
-44793267 133583991
104449468 -71907441
74219151 -138641903
18622765 -60852013
-19781724 47622956
58120233 -141231642
-62852805 133530656
-6980476 102250319
100532075 -118053636
-21092448 46456131
-3942790 89520583
-29682554 128245698
-16758670 50660567
8388554 -88915439
-101856812 97750635
-31040059 39797918
25387671 -53622489
-55634903 134341354
69215265 -42430292
66767168 -140515935
-6343343 68272240
-85434383 122999865
-4042242 90405578
13486421 -113336806
-101546416 69936839
105489920 -107857382
14279682 -114878279
96414573 -123634805
9997994 -78443521
-33310897 38715798
-21062780 46481643
-35075789 130790503
100449685 -118182820
-3777357 87533391
-41685996 132920342
25017659 -128514329
12062295 -72288763
107975109 -97219188
-6053185 69151533
63308012 -41480040
84259197 -133986383
8351578 -89939059
8449332 -87850603
-103517171 80375052
90983600 -129101080
73593221 -138847040
52496611 -41574123
102061140 -66983143
-26519051 126360649
55942126 -41289387
-23142075 123972216
92817742 -127438877
-33824053 130268982
-52148477 134356278
-8951931 106731332
-78291879 127901471
-66675028 132660848
17704162 -62091979
106638244 -104146537
-100083606 65765133
52824113 -140927492
32187209 -48614025
-92484857 115907631
-94233220 113624462
-101159706 100047104
73497162 -138877716
-59852464 34762401
67410707 -140401493
9503753 -101980284
107721513 -99039665
8888732 -98653750
-27308632 126861044
-8418427 63129042
-56079183 34437371
-101601554 70120861
-102182842 72244568
30662752 -49587719
-101395836 69446857
-32147300 39252766
-83316294 124648264
9218243 -100589364
35739410 -46628898
108227081 -94513242
-50450914 134275435
-103557421 80912218
-4519871 93517404
-24702342 43631384
-103399106 79107903
-99722523 103870268
95318284 -124880669
49827231 -140502990
-21562022 46057656
-79056993 41299555
35199055 -46906774
103475249 -112734310
-22706760 123632630
47988041 -140149824
-14208715 115074809
61102155 -41308741
-80329825 126684157
68394924 -140209598
14655811 -115566052
-89457777 119305208
83557915 -134403821
-85775245 46042301
13741560 -68617108
45570944 -42888721
-61711669 133730020
-102461994 73420555
8519134 -95520918
-3691652 83202759
-45290949 133671261
45341706 -42949828
37578315 -136720189
21909821 -56979849
-38182310 131917970
73661830 -43639205
-4772169 73975854
-101463093 69663503
-65927511 132855898
32636957 -134122871
-71588977 131061463
-4110170 77818404
9981581 -78505720
82777158 -134850606
-4067039 78152912
-76391557 39836600
17381593 -62547084
29227179 -131886046
-101308277 69169886
68400823 -140208386
16144352 -64400007
100738679 -117725796
-96433301 110301997
95148419 -125066427
-4213549 77079805
51661668 -140784787
-70773285 131366311
95838441 -124299928
98179693 -121437879
76860606 -137673426
9211367 -81911005
-5494051 97733565
74410334 -43886786
-15015226 116084648
-18485346 48862193
-103474615 79876665
-84102181 44701504
-59694135 134016389
80237205 -136179706
-3991574 89972443
26607939 -52587533
36519216 -136221855
-100663388 67281298
105302812 -108378854
-85355247 45694542
-12916155 55423006
34137162 -47477427
-78821503 41161688
-52347048 34397384
72135430 -139289819
35780623 -46608044
-87067414 121597055
67633972 -42104448
88734363 -51534874
9514505 -80435188
69844232 -139889417
24201547 -127768622
-100663533 67281696
12540439 -111303760
8730781 -97511898
-4391436 92797183
-100757947 101215932
-27226553 41949357
-17658297 49701213
-103619745 81970940
-47647849 134014737
-72587504 38093356
-103628319 82155820
-86735385 46867060
36695384 -136306855
-3732793 82035226
28672238 -50981275
-90849887 50939638
-6424821 68034759
-16986208 50413112
-61095676 133826361
8338110 -90661625
91654083 -53951691
-86012444 46242148
41825247 -138427789
9842552 -79046188
95268284 -124935542
26283002 -52856607
8465207 -87624031
85579530 -133157366
9027656 -82937522
-28333596 41278959
76446947 -44627895
-3790638 87736940
100805524 -117618507
108093543 -96138533
-32972787 38868107
-3710620 82568287
81942562 -135307903
-16480531 50967648
63830460 -140929168
48244981 -140203466
39946543 -137728016
67060139 -140464911
-103620435 81985293
73870306 -43706849
-32055671 39296597
23453838 -55408722
-24920041 125281889
-96476297 110230948
65970595 -140645618
-43081634 35515356
-76385499 128925831
-50190439 134257906
-83585959 124448387
-15680746 51880072
8545128 -86651384
8377918 -89157389
-12021352 56725731
-4292185 92193834
-68823611 36728813
-84887663 45316333
-11555441 57440723
69778615 -42559314
-92094893 52377689
-39717710 36368049
41951022 -138471599
-101263425 99729205
107901859 -97798089
95534017 -124642016
-83849382 44508936
12003001 -72434250
-3681554 84992516
-103541283 88071952
-87419698 121277967
-81627381 125838336
-57991357 134193256
-5739208 70170819
32718200 -48292762
-47844404 134038221
-23313467 44655530
-9776306 60449378
-26347807 126249369
-36564921 131359980
-91489254 117095018
-50286478 134264528
106776471 -103617977
-92745430 53175182
-90165705 50194402
-42680890 35603981
-3997348 90023504
-9442106 107686398
-92114394 52401114
-44427473 35242807
23758532 -127349658
-9340288 61267226
-96272893 58194601
16774873 -63433401
8406547 -93908664
-41444041 35899416
-101875240 97684056
106296138 -105364173
107552683 -100041501
8356735 -89753106
65964460 -41817640
103808941 -112019299
25341841 -53662687
-4795189 73868224
12858733 -112014842
108072232 -96350052
107858015 -84343222
56762995 -141207389
-79294000 41440060
85087081 -48990909
-76187609 39733188
-65272668 35742659
-18178579 49168801
-34566430 38176256
-60455617 133918153
-4107643 77837526
13634219 -113634180
26081177 -53026079
-87417248 47479135
-38330758 131966102
-4377032 76046940
-9318568 61308939
72198924 -43192665
51298525 -41729678
-7220633 65893682
-74065130 130034427
44555802 -139295994
-49931583 134239133
-73227634 130399275
13365560 -69374325
-8459496 105717551
107079703 -102364931
-15712671 116916560
-71304779 131169516
-72647321 38117838
-76817017 40056127
11346168 -108321959
88668246 -130979875
79946356 -46143933
48033693 -140159457
-12416352 112619591
-103465398 88983465
-78259220 40839417
-33079568 129939664
18853698 -60552497
17797495 -120501923
-26723245 42267298
-101513560 69828384
12706058 -111677465
-83064687 124832246
15664487 -65169876
9312239 -101069943
-51022230 34450242
58742197 -41233762
20961459 -58017208
-3962151 89704554
108273357 -93711230
-4431560 93028834
101105736 -65335156
55565318 -41308865
56105162 -41281843
107488792 -100391281
106822236 -103437589
-95795791 111325445
10926163 -107119704
70079288 -139833217
66755250 -41946233
106829280 -79054631
-102880915 93270863
-13971800 53991620
-77304864 128445098
34934356 -47045944
-41496312 132873087
9365777 -81131146
-103324189 78443687
-6141076 99879995
-59157107 134078536
-84307308 44859637
-7761067 64589428
95975460 -58320328
96407851 -123642702
58881957 -41235096
96519877 -58953562
105253873 -73951889
9098265 -99936021
54121389 -141054253
52197990 -140854062
-33267011 38735397
104537578 -110345145
51947091 -41641816
-3704705 82739581
-73355699 130344661
-10995468 110422303
-13770624 54256302
-64395623 133217446
8406973 -88547592
-30477972 128671522
11626918 -73389217
-48496942 34648809
-42904876 133205322
8378684 -89139041
100347856 -64122920
106901239 -79344867
-97603115 60493137
88663816 -51480950
-35278583 130871271
38568867 -137159281
35190415 -46911285
103830230 -111972664
68016921 -140285722
-59644780 134022346
-97143958 59667468
-3710036 86174861
108315700 -92615995
-7983085 64082057
-55628273 34417730
-102678550 94325845
-101629118 70213832
104830485 -72843073
78875690 -136818001
62569001 -41411705
21908837 -56980896
-3722423 86491653
-84460707 44979005
-87045694 121616529
47752058 -42364891
95709672 -124445400
-27731114 127120396
-40599084 132638839
98136884 -121494265
-65694572 35845177
22421558 -56443670
108120652 -86608845
19441598 -59810541
-93368940 114787125
-4432096 75727456
-8560615 105930536
-62091763 35092704
-9471483 61017272
-40614593 36116304
-60476513 133915290
105424808 -74423284
-70713944 131387859
-28188188 127394550
28079948 -131039647
-100611265 101620604
-96170777 58029212
-21383789 46207732
9264138 -100827136
-77152029 40232639
-100395000 102198287
108326725 -92133782
52056468 -140836355
9486196 -101900208
-15791383 117008179
-97952316 61146784
-94477191 113283123
-60974924 133844325
-83978200 44606745
-3736668 81954087
68045814 -140280011
-77238229 40278582
101760937 -116014666
18569742 -60921448
32804630 -134222888
-12360085 112537202
81487430 -135548672
18502048 -61010463
83734994 -48164331
66036940 -41828885
8617981 -96545514
18009511 -61670868
92543657 -127697948
99375624 -119791782
105470578 -74552065
66345447 -41877970
-13703900 114414442
-25757607 125858135
-88982780 48974046
69947536 -139864866
92070322 -54327925
74894690 -138410189
104353515 -71680045
9258870 -100800156
-103566401 81043901
-79477606 127209217
-102252805 72527721
18430782 -121359580
46992285 -139928572
-4316890 76410937
13384218 -69335980
8613316 -85962519
82807133 -134833795
91218295 -53566742
-4453922 75604141
23679580 -127273910
103298665 -69363041
48407261 -140236621
-7645049 64860809
11474221 -73794186
60900271 -41297959
-10032228 59985735
-14341707 115245056
-3710236 86180430
29131599 -131817443
-56267122 134312581
-37552714 131708116
20218144 -123591279
92820313 -55027776
-39675985 36380200
102474670 -114719903
82911672 -134774956
-6467853 67910881
10486478 -76722355
-25789828 125879805
-101319168 69204037
9449140 -80735109
-79899508 41807123
78950297 -45679896
36858036 -46079625
-38535280 132031577
-100908004 100790269
-12126470 112191308
8428739 -94294697
-33113482 38804361
61637107 -41341270
108296808 -93182305
-54877308 134365281
41632166 -138359803
-28935190 40930813
-51831321 134345565
108299297 -89346743
-4049523 78294093
79024168 -136750806
-4816878 94991480
-5275133 96916896
65785270 -41790305
-40019611 36281294
8818433 -84294961
-37526839 37060051
-88989999 119778096
19368406 -122562851
74476023 -43909141
107900826 -84658323
105307734 -108365365
10162169 -104625844
18536275 -60965404
-3906984 79597833
-20757931 46746145
-93653420 114412148
8611257 -96482218
82820326 -47637817
99565359 -62947026
13353368 -113064779
21146437 -57810230
-12568362 55918922
101592749 -116307083
-18260314 49086588
41729811 -138394296
17964207 -120731436
20396447 -58664064
22882958 -55973586
96438368 -123606820
43338961 -138930354
-41667626 132915800
41973141 -43984941
-84496658 45007119
-61840331 35050440
-4993906 72983242
-22830997 45029006
-4600574 74817976
-99103941 105271431
89210017 -51903685
-39527860 132335685
98173820 -61018580
-3764066 81443799
10575362 -76432414
-82196329 43310299
-100810480 67691043
-9639158 108056626
84760624 -133678289
-3785329 87657091
78701995 -45568335
24400530 -127953507
-9058531 106943381
-85179238 123208260
108134043 -95708816
63831155 -140929091
-54453539 134373653
-6160074 68821207
27496930 -51874481
-26251713 126186487
60586507 -41282828
104909410 -109420311
67093384 -140459008
-4608634 93982647
-5949964 99281142
55215169 -41329522
81585302 -135497402
80514946 -136043302
47070064 -139946623
-102900235 93163278
31271947 -133274926
107839454 -84211081
-6157595 99930565
103505863 -69794284
107072478 -80067686
-6290242 100330204
8464671 -94832759
12405047 -71472227
-103624822 82078621
-9522196 107837779
74367894 -43872397
90355681 -129633492
-4127283 91068947
15304382 -116695378
104875721 -109506342
-103166026 77244176
20548901 -58487307
60824005 -41294100
-72795487 130580472
-4632047 74657829
60188594 -41266478
12153425 -110396535
16514662 -63826421
29125623 -50651063
9085365 -82601479
10796561 -75736602
8477667 -87455756
108332889 -91675520
-25512865 125692339
-92047909 116437972
17386120 -119923583
-7093589 66216207
-32273628 129566633
-5407628 97417719
103423470 -112842675
-9163187 61610189
25846530 -129238796
-29342723 128058088
31909591 -48785602
-100168602 102780898
-59287322 134064008
-92114517 116358081
-66617356 36083007
95669040 -124491066
-8531905 105870328
41690007 -44083942
-91147815 117485530
-40447987 132597636
12192003 -71974969
-11225845 110796174
-52174911 134357080
90631139 -129402118
94789710 -57011588
98783663 -61840030
-6807409 66967991
-103595799 87242053
38831559 -137271449
93587529 -55774395
79729709 -46040729
104764419 -109787308
100877611 -64962082
20262521 -58820726
83010304 -134719138
16830158 -119113280
68574389 -42291813
26597287 -52596280
-89855257 118893313
68228948 -140243397
-54661195 134370000
-77439496 128372645
14595820 -115458056
10304087 -105126177
-39293842 132266014
40566642 -137968497
67412190 -42062986
-76070776 39674475
37943530 -45579131
-46810596 34853501
-77566478 128303822
56952973 -141213003
12614134 -70993139
14332946 -67486956
36294542 -46351989
-49465836 134201949
106176547 -105763881
100932836 -65051727
-7646497 64857396
87449117 -131881331
-74521696 38931548
98813344 -61880919
105862236 -75703158
-102947053 92896749
67820248 -140324132
34622098 -135251475
37247792 -136567914
20200871 -58893283
107304120 -81132044
108323041 -90145219
71101337 -42889399
73570540 -138854302
24119743 -127692027
75381172 -44227602
-61409863 133778206
-53217507 34381790
10181624 -77768587
-101590023 98677181
-79247550 41412384
95499623 -57783932
-100833450 101003286
8800369 -98039060
-26769961 42237433
-4092677 90807403
107048179 -79962139
91773587 -54058843
11379869 -108414340
-4879907 95277641
-8948093 62035692
-103128909 76991324
-3820652 80602780
-6368302 68199080
-102623175 74164481
11851513 -109652171
-5495576 97739065
-47742661 134026163
62468729 -41403285
-73622887 130229355
-65360943 132995555
