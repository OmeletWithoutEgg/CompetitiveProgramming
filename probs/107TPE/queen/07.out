2328814.54 -3425723.26
