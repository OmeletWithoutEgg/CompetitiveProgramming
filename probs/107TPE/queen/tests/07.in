270
-50149481 23005142
-103878297 87386403
-128692360 56183415
-50518059 22453848
-118502707 18013571
-128054199 35746942
-47388330 27846006
42040991 -85480408
-88497618 91123469
70651543 -53908542
-88678548 3511858
69696488 -128070610
57872512 -57541148
94610540 -109883683
-112722483 12581944
42233343 -84011229
-43518575 54775105
43636275 -77621944
-68879371 87512782
44927090 -108087694
-112693824 12559581
-47551010 27516994
-109730167 84182513
81830289 -124800302
-104142106 87266399
-55026950 77759743
-121630911 21981291
51814652 -119327551
-117972986 17426921
84958298 -59332004
53464504 -60965282
92410858 -67925251
48975009 -115729298
78680732 -55611893
79755483 -56083931
-73051923 89104634
95933413 -106605423
-72949041 5554135
-126941031 62407147
-129585247 46834542
-89148220 91084128
86072133 -121712174
-102185983 6519085
58479932 -124805141
-50908225 21889244
-43264250 53089203
42203715 -97762018
-88284556 91134195
70559114 -128071529
-129580450 48121713
-52523230 74890720
-126923493 32169889
-46870895 65683968
-128376463 57611122
98198405 -97016411
54728491 -59836012
67029562 -127851069
45182126 -73264713
57082850 -123923753
71506035 -128033189
-57736941 14256021
49797099 -65100950
-77369312 4369971
-103695314 7156997
-65434361 8823600
42250238 -98081010
86435972 -121385826
-45268270 61653167
98415686 -87051912
-53742948 76351494
66263547 -54251246
52265159 -119813486
-51262181 73232507
-77755529 90335240
60363174 -56170727
64398826 -54670067
41729446 -90384341
-126965363 32285602
-55092817 16796643
49639184 -116664155
46930752 -69597719
98562418 -92148900
-116434889 78788355
-61222375 83142410
-125124802 66700493
-58277542 80838837
45851676 -110232379
83993744 -123380582
-55142117 77880185
-122009077 72094097
90163643 -64651972
98117321 -84351300
-102534205 87965105
-43224321 41848999
61177419 -126178855
-48080075 68133499
45727067 -109961950
-60279731 82449001
53831814 -60623557
98000396 -83561478
-116281275 15686837
-118067848 17530419
42060699 -85311852
-125555285 65789929
-107089091 8850665
-115866070 15288185
79112767 -126183974
42119898 -84833897
-117299065 77914997
57084139 -123924624
-129562082 48824242
-129512761 49893213
-107941534 9336705
-129560837 45765012
-127522645 33933385
-46977389 65915893
-128887222 39457388
-118302154 17788948
-58631426 13487542
52964141 -61449657
-120367714 74361197
76250998 -127211337
96618164 -77462486
-128512936 37603965
65897657 -54320318
95687164 -107279610
44024495 -105599594
92153258 -114469880
98265875 -96455206
-107983895 9361532
-93873607 4128246
72465340 -127952225
84432487 -58923425
42293096 -98362226
44510823 -74979020
98066965 -97978356
63669101 -127098260
95449224 -74078396
98443561 -94569239
77851223 -55289470
-111626078 11753274
-98281297 5169382
89791396 -117798936
51053953 -63519128
89409482 -63711661
56038514 -58796300
49324067 -65749502
-129090635 40687461
93886179 -70582418
-64597724 85326429
82259127 -124542537
96262908 -105643911
73508920 -127815672
97450535 -101322881
-43008685 50599987
88344070 -119484297
-122545665 23343809
-60488551 82606008
-126309170 64049275
-53244380 18855456
81651743 -124904323
88505153 -119307912
-128380477 57594288
-43032328 50904231
45558936 -109588588
98576273 -91058491
-96349023 4652377
-123390531 69916772
65364988 -54432248
64574873 -127354781
42999447 -80011603
-46077690 30792972
-56150746 78897032
-113036692 81795706
71060476 -53921779
-47236104 66465108
-120664557 20652240
74870877 -127560527
-111320170 11531629
-122612313 71177861
-44361842 58659259
92267935 -67692376
-118661388 76431856
-106246576 8395030
97631971 -100468998
51883721 -62574763
-61290630 83191113
-126758048 62903200
-88209670 91137712
-97959070 5076436
96343915 -76582409
45921720 -71596007
-118285080 17769968
88097704 -119749006
-73480405 5384771
-89464867 3564000
81504006 -124988948
-109370727 10210839
-96125598 90026610
79061192 -55771886
41780732 -88681775
-118844083 76222174
94799018 -109462055
47115363 -112717529
-118576898 76527922
64678061 -127381455
-54347113 77029515
-107338358 8990143
64432332 -54661062
98139370 -97466717
98573234 -90442371
51350431 -118804960
98123360 -84394760
92160295 -67519491
-120385045 74338785
87627593 -120237673
48947081 -66289596
-98347959 89436431
44734880 -107596127
-122921353 70688389
-53277712 18815926
-129055359 40457857
-106438168 86128854
51448327 -118917071
-73104385 5503835
76755666 -54917774
85352386 -122326578
69749453 -53906492
-46823713 29045312
96907875 -78470246
-129357062 42792146
-43829953 38211395
55125199 -59507738
96509313 -77105396
92625508 -113695782
46463681 -111490188
-80894931 3776653
97024122 -103076619
96456170 -76934872
58896772 -56933334
-63621860 9885227
61531939 -126329381
-47004647 65974693
44764865 -107674013
-106688344 8630927
48144242 -114462718
98533793 -88961122
-63440038 9998366
-57747797 80378944
-85750552 3445006
79914004 -125819271
47506708 -113406826
-87342098 91168894
-98737892 5305774
92352944 -114147719
-105956827 8243862
-79627459 3955202
80221185 -125670078
-116872940 78351801
85591201 -122127200
-104957025 86882029
97168848 -79463776
57329906 -124088630
42421950 -82830905
-62897272 10343613
-47764905 27094586
70821569 -128065030
61825748 -55529069
-62523209 84037163
-48268077 26142824
