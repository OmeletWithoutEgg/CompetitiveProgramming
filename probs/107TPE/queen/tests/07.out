1458007.94 27243316.16
